po/common/fr.po

```diff
@@ -1134,7 +1134,7 @@ msgstr "Les produits qui possèdent le label %s"
 
 msgctxt "labels_s"
 msgid "label"
-msgstr "label"
```

Comment:
that looks like a bug

Reply:
why ?

Reply:
I'm fixing it, but that should have no impact